lib/environment.rb

```diff
@@ -318,22 +316,17 @@ def version_cmp(version, user:)
 
       major, minor = parse_version(version)
 
-      norm_version = @major_version == '1' ? '3' : @major_version
-      if ( major == '1' )
-        major = '3'
-      end
-
-      # presently handle only major ver `4`, `3`, `1` for OCP and OKD
-      bad_majors = [@major_version, major] - [1, 3, 4]
+      # presently handle only major ver `4`, `3` for OCP and OKD
+      bad_majors = [@major_version, major] - [3, 4]
```

Comment:
I'm not sure we need this check anymore. I'd say, lets remove it altogether for the sake of simplicity.

Reply:
@akostadinov  Updated.